### DISCUSSION TITLE
Species Images

### DISCUSSION THREAD

**Discussion by Schiano-NOAA** [nmfs-ost/asar#94] [General]
We are currently on the hunt for NOAA illustrations of species. We have started to compile them into [inst/resources/spp_img](), but are currently missing some. If you do not see a specific from your region listed, please let us know in this discussion post or email Sam Schiano (samantha.schiano@noaa.gov) the illustration to be added to the repository.

Here is a list of images we know we need:
### Bering Sea and Aleutian Islands
Note: The remaining fish in this category are not available in the NOAA Species Directory.
- [ ] Alaska Plaice
- [X] Arrowtooth Flounder*
- [ ] Atka Mackerel*
- [ ] Blackspotted and Rougheye Rockfish*
- [X] Flathead Sole*
- [ ] Forage Fish (?)
- [X] Greenland Turbot
- [ ] Kamchatka Flounder
- [ ] MS assessment (walleye pollock, pacific cod, arrowtooth flounder)
- [X] Northern Rock Sole
- [ ] Northern Rockfish*
- [ ] Octopus*
- [ ] Other Flatfish
- [X] Pacific Ocean Perch*
- [X] AI Pollock
- [ ] Bogoslof Island Pollock
- [ ] EBS Pollock
- [ ] Sharks*
- [ ] Shortracker Rockfish*
- [ ] Skates*
- [x] Yellowfin Sole
- [x] EBS snow crab
- [ ] Red king crab (mult. stocks)
- [ ] Blue king crab
- [ ] Tanner crab
- [ ] Golden king crab (mult. stocks)
- [x] Scallops

### Gulf of AK
Note: The remaining fish in this category are not available in the NOAA Species Directory.
- [ ] Alaska Sablefish
- [ ] Deep-water Flatfish
- [ ] Demersal Shelf Rockfish
- [ ] Dusky Rockfish
- [X] Northern and Southern Rock Sole
- [ ] Other Rockfish
- [X] Pacific Cod
- [X] Walleye Pollock
- [X] Rex Sole
- [ ] Shallow-Water Flatfish
- [ ] Thornyheads

*fish species that are evaluated for both regions

## NEFSC

- [X] Acadian redfish
- [X] American Plaice
- [X] Atlantic bluefish
- [X] Atlantic cod
- [X] Atlantic halibut
- [X] Atlantic herring
- [X] Atlantic mackerel
- [X] Atlantic sea scallop
- [X] Atlantic surfclam
- [X] Atlantic wolffish
- [X] Black sea bass
- [X] Butterfish
- [X] Deep sea red crab
- [X] Golden tilefish
- [X] Haddock
- [X] Longfin squid
- [X] Monkfish
- [X] Ocean pout
- [X] Ocean quohog
- [X] Pollock
- [X] Red Hake
- [X] Scup
- [X] Shortfin squid
- [X] Silver Hake
- [X] Skate complex
- [X] Spiny dogfish
- [X] Summer flounder
- [X] White hake
- [X] Windowpane flounder
- [X] Winter flounder
- [X] Witch flounder
- [X] Yellowtail flounder

### Groundfish

Note: The remaining fish in this category are not available in the NOAA Species Directory or [PFMC documents](https://www.pcouncil.org/stock-assessments-star-reports-stat-reports-rebuilding-analyses-terms-of-reference/groundfish-stock-assessment-documents/).

- [x] Yelloweye rockfish
- [x] Widow rockfish
- [x] Sablefish
- [x] Shortspine thornyhead
- [X] Petrale Sole
- [ ] Copper Rockfish
- [x] Black Rockfish
- [X] Rex Sole
- [x] Canary Rockfish
- [X] Pacific Whiting (hake)
- [ ] Quillback
- [x] Lingcod
- [ ] Vermilion Rockfish
- [ ] Sunset Rockfish
- [ ] Squarespot Rockfish
- [x] Spiny Dogfish (identical to NEFSC's spiny dogfish)
- [X] Dover Sole
- [x] Big skate
- [x] Cabezon
- [ ] Cowcod Rockfish
- [ ] Gopher Rockfish
- [ ] Black and yellow rockfish
- [ ] Longnose Skate
- [x] Arrowtooth Flounder (identical to AFSC's arrowtooth flounder)
- [ ] Blackgill Rockfish
- [x] Blue Rockfish
- [x] Deacon Rockfish
- [ ] Chilipepper Rockfish
- [ ] China Rockfish
- [ ] Darkblotched Rockfish
- [ ] Kelp Greenling
- [ ] Aurora Rockfish
- [ ] Blackspotted Rockfish
- [ ] Rougheye Rockfish
- [x] Pacific sanddab
- [ ] CPS
- [x] Pacific Sardine
- [x] Northern Anchovy
- [x] Pacific Mackerel

## PIFSC
Note: The remaining fish in this category are not available in the NOAA Species Directory or [Google Drive folder](https://drive.google.com/drive/folders/1r1m_0RXjxe0LH8_tBaCdRMibwAl5S6o6?usp=sharing).
- [x] Aprion virescens/Uku/Green Jobfish/Grey Jobfish
- [x] ~Epinephelus~ Hyporthodus quernus/Hapu'upu'u/Hawaiian Grouper
- [ ] Etelis boweni/Bowen's Snapper/Giant Ruby Snapper
- [x] Etelis coruscans/Onaga/Longtailed Red Snapper
- [x] Etelis carbunculus/Ehu/Squirrelfish Snapper
- [x] Lutjanus kasmira/Ta'ape/Bluestripe Snapper
- [x] Lethurinus rubrioperculatus/Spotcheek Emperor
- [x] Aphareus rutilans/Lehi/Silverjaw Snapper/Rusty Jobfish
- [x] Caranx lugubris/Black Jack
- [x] Caranx ignobilis/Giant Trevally
- [x] Pristipomoides auricilla/Goldflag Jobfish
- [x] Pristipomoides filamentosus/Opakapaka/Hawaiian Pink Snapper/Crimson Jobfish
- [x] Pristipomoides flavipinnis/Golden Eye Jobfish
- [x] Pristipomoides sieboldii/Kalekale/Lavender Snapper
- [x] Pristipomoides zonatus/Gindai/Brigham's Snapper/Oblique Banded Snapper
- [ ] Ranina ranina/Kona Crab
- [x] Variola louti/Yellow-edge Lyretail Grouper

*clarification is needed on the acronyms and common names for these species*
Is there an images with multiple species for the big7?

## SEFSC
Note: The remaining fish in this category are not available in the NOAA Species Directory.
- [x] Wahoo
- [x] Dolphinfish
- [ ] Red drum
- [x] Blueline tilefish
- [x] ~Atlantic~ Golden tilefish
- [x] Vermilion snapper
- [x] Gray triggerfish
- [x] Spanish mackerel
- [x] Black sea bass
- [x] Atlantic menhaden
- [ ] Scamp
- [ ] Yellowtail snapper
- [ ] Mutton snapper
- [X] Red snapper
- [ ] Red porgy
- [x] Cobia
- [x] Greater amberjack
- [x] Black grouper
- [ ] Snowy grouper
- [x] Red grouper
- [x] Gag grouper
- [x] Shrimp (white, pink brown)
- [ ] Yellowedge grouper
- [ ] Gray snapper
- [ ] Gulf menhaden
- [x] King mackerel
- [x] Spiny lobster
- [ ] Stoplight parrotfish
- [ ] Queen triggerfish
- [ ] Hogfish

## SWFSC (some in collab w/ NWFSC)

- [ ] Groundfish

## HMS
Note: The remaining fish in this category are not available in the NOAA Species Directory.
- [x] Yellowfin tuna
- [x] Bluefin tuna
- [x] Skipjack
- [x] Swordfish
- [x] Blue marlin
- [ ] White marlin

**Comment by kellijohnson-NOAA** [comment]
@Schiano-NOAA I suggest reaching out to John from our communications department and Jeff from ST4 to see if some of these can be found somewhere already. Then @MOshima-PIFSC can probably get some of PIFSC ones for you. @melissamonk-NOAA can probably get all SWFSC ones that are in the NWFSC list. @KyleShertzer-NOAA will know who to contact for the SEFSC. @JaneSullivan-NOAA might be able to get epic pictures of some from when she was out on the survey or know where pictures exist for the AFSC. Also @MargaretSiple-NOAA is a great AFSC resource from the survey team. I see that none of these people are following this repository as of yet, which is why you are tagged 😃!

**Reply by Schiano-NOAA** [reply]
Thanks so much Kelli! I will start with comms then work my way around to others if there are still more needed images. @MOshima-PIFSC already provided me those from the species she works on :)

**Comment by iantaylor-NOAA** [comment]
@Schiano-NOAA, I would just rename the NWFSC section to NWFSC/SWFSC or West Coast as the list represents species assessed by both centers either in collaboration or separately. 
It looks like the checked boxes in the NWFSC list are images that are found at https://www.fisheries.noaa.gov/species-directory but that source includes many of the unchecked boxes as well. Let me know if you want help gathering them (although I'm guessing someone from communications would be better at this).

**Reply by Schiano-NOAA** [reply]
Good to note. I had figured as much, but wasn't 100% sure. I will reach out to comms and see how they can help us on this. I'll follow up with everyone on the discussion afterwards. Thanks for the input!

**Comment by JaneSullivan-NOAA** [comment]
@Schiano-NOAA It looks like you want only illustrations? Probably not survey photos... Is that correct?

**Reply by Schiano-NOAA** [reply]
Yep! only illustrations preferably and in png format...although we can change that if need be

**Comment by MOshima-PIFSC** [comment]
@Schiano-NOAA I clarified the species names for PIFSC. I included the scientific name and common name(s). Most of the species here we call by their local names (which can be different between Hawaii/American Samoa/Guam) and from their English common name (e.g. onaga vs longtailed red snapper). I believe I got all of the species we assess currently between Hawaii, American Samoa, Guam, and CNMI, however @marcnadon would know best probably. There is an image of all the [deep7 species](https://media.fisheries.noaa.gov/dam-migration-miss/md_IeLgUvKRP161.jpg?1547205937) together but I'm not sure where the original is. Let me know if there's more I can help with!

**Comment by KyleShertzer-NOAA** [comment]
The list for SEFSC looks pretty good. I assume Atlantic tilefish refers to golden tilefish (which is actually just called tilefish)? You might add to the list dolphinfish and wahoo, which are in the same FMP. Also, do you want to include HMS species (international and/or domestic)? If so, I can put you in touch with someone from the HMS branch for a good list of species.

**Reply by Schiano-NOAA** [reply]
@KyleShertzer-NOAA Thanks for clarifying that Kyle! Yes, we are referring to golden tilefish there. We will add wahoo and dolphinfish to the repo and I can reach out to Dean Courtney for the HMS images 😄

**Comment by MargaretSiple-NOAA** [comment]
I wrote a couple of these in email form to Sam but wanted to add here because there are another couple of AK stocks I neglected to mention. They aren't on the AFSC site with the other SAFEs but are listed [here](https://www.npfmc.org/library/safe-reports/): 

### BSAI:
- EBS snow crab
- Red king crab (mult. stocks)
- Blue king crab
- Tanner crab
- Golden king crab (mult. stocks)

...and there are scallops (one assessment) but I know almost nothing about that assessment, its frequency, species covered, etc.

**Comment by marcnadon** [comment]
@Schiano-NOAA and @MOshima-PIFSC 

For the PIFSC species, images can be found here:
https://drive.google.com/drive/folders/1r1m_0RXjxe0LH8_tBaCdRMibwAl5S6o6?usp=sharing

Also, Epinephelus quernus is now Hyporthodus quernus.

**Reply by sbreitbart-NOAA** [reply]
Thanks! These images have been added to the repo.

**Reply by sbreitbart-NOAA** [reply]
@MOshima-PIFSC @marcnadon Can you please check that these species names are correct? https://github.com/nmfs-ost/asar/pull/103/commits/38a16822cde48daa83621c38f7eb1725831c7229
